Tools/scripts/cygwin_build.sh

```diff
@@ -8,22 +8,29 @@
 
 set -x
 
+# TOOLCHAIN=i686-pc-cygwin
+TOOLCHAIN=x86_64-pc-cygwin
+GPP_COMPILER="${TOOLCHAIN}-g++"
+
+$GPP_COMPILER -print-sysroot
+
+SYS_ROOT=$($GPP_COMPILER -print-sysroot)
+echo "SYS_ROOT=$SYS_ROOT"
```

Comment:
Given that print-sysroot is coming up blank, and the SYS_ROOT env variable is unused in this script, do we need these lines?

Reply:
@robertlong13 not with the current formulation, but if someone did want to go back to the x86 version (presumably with their own custom repository) this might be useful to them.  At least in the short term we can keep this as doing no harm, I reckon.